src/Discord.Net.Core/Entities/Guilds/IGuildIntegration.cs

```diff
@@ -1,17 +1,34 @@
-﻿using System;
+using System;
 
 namespace Discord
 {
     public interface IGuildIntegration
     {
+        /// <summary> Gets the integration ID. </summary>
+        /// <returns> The unique identifier value of this integration. </returns>
```

Comment:
For `ulong` identifiers, the preferred format is `An <see cref="ulong"/> representing the snowflake identifier of the...`